README.md

```diff
@@ -1 +1,55 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Leaderboard
+
+> This is a Leaderboard Website
+
+## Built With
+
+- HTML
+- CSS
+- JS
+- Linters
+- Webpack
+- ES6
+
+## Getting Started
+
+- To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Html & CSS basics
+- Git & Github
+- SSH key for github linked to git on local environment
+- Code compiler
+
+### Setup
+- Click on the green button on the top right corner with the text code
+- Click on SSH
+- Copy the text that appears below
+- Type "git clone" in your terminal and past the copied text next to "git clone" and press enter
+
+### Run tests
+
+- For tracking linters errors locally you need to follow [these steps](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
+- For installing Webpack follow [this tutorial](https://webpack.js.org/guides/getting-started/#basic-setup)
+- For setting up Webpack HTML plugin follow [this tutorial](https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin)
+- For setting up Webpack CSS plugin follow [this tutorial](https://webpack.js.org/guides/asset-management/#loading-css)
+- For setting up a LiveServer follow [this tutorial](https://webpack.js.org/guides/development/#using-webpack-dev-server)
```

Comment:
- [ ] Instead of these steps, Please add the instructions on how to install project dependencies (libraries) `npm install`, run webpack build `npm run build`, and run webpack dev server `npm run start` To make your readme file more professional and user friendly.